fe/fe-core/src/main/java/com/starrocks/sql/optimizer/OptimizerContext.java

```diff
@@ -75,6 +76,8 @@ public class OptimizerContext {
 
     private boolean isShortCircuit = false;
 
+    private List<IsNullPredicateOperator> pushdownNotNullPredicates = Lists.newArrayList();
```

Comment:
the nullable info isn't global. If the predicate is deduced first and then push down outer join. like:join->outer join->join, will take bugs
BTW, push down worked by Top-Down, shoulde be not question, but you'd better test it

Reply:
BTW, add comment

Reply:
fixed.